docs/modules/globalfee.md

```diff
@@ -29,41 +31,40 @@ A global fees list must meet the following properties:
 - fees have to be alphabetically sorted by denom; 
 - fees must have non-negative amount, with a valid and unique denom (i.e. no duplicate denoms are allowed).
 
-
 There are **two exceptions** from the global fees rules that allow zero fee transactions:
 
-1. Transactions that contain only [message types that can bypass the minimum fee](#bypass-fees-message-types) may have zero fees. We refer to this as _bypass transactions_. Node operators can choose to define these message types (for each node) via the `bypass-fee-message-types` configuration parameter.
-
-2. One of the entries in the global fees list has a zero amount, e.g., `0uatom`, and the corresponding denom, e.g., `uatom`, is not present in `minimum-gas-prices`.
+1. Transactions that contain only message types that can bypass the minimum fee requirement and for which the total gas usage of these bypass messages does not exceed `maxTotalBypassMinFeeMsgGasUsage` may have zero fees. We refer to this as _bypass transactions_.
 
-Additionally, node operators may set additional minimum gas prices which can be larger than the _global_ minimum gas prices defined on chain.
+2. One of the entries in the global fees list has a zero amount, e.g., `0uatom`, and the corresponding denom, e.g., `uatom`, is not present in `minimum-gas-prices` in `app.toml`, or node operators may set additional `minimum-gas-prices` in `app.toml` also zero coins.
 
+### Global Fee Params: `BypassMinFeeMsgTypes` and `MaxTotalBypassMinFeeMsgGasUsage`
 
-### minimum-gas-prices
+Bypass minimum fee messages are messages that are exempt from paying fees. The above global fees and the below local `minimum-gas-prices` checks do not apply for transactions that satisfy the following conditions:
 
-The `minimum-gas-prices` config parameter allows node operators to impose additional requirements for minimum fees. The following rules apply:
-
-- The denoms in `min-gas-prices` that are not present in the global fees list are ignored. 
-- The amounts in `min-gas-prices` are considered only if they are greater than the amounts for the corresponding denoms in the global fees list. 
-
-## Bypass Fees Message Types
+- Contains only bypass message types defined in `BypassMinFeeMsgTypes`.
```

Comment:
**Transaction** contains only bypass message types defined in `BypassMinFeeMsgTypes`.